scripts/build-linux.sh

```diff
@@ -166,11 +166,16 @@ if [ ${CPU} = vexriscv ]; then
 		RAM_BASE_ADDRESS=${RAM_BASE_ADDRESS::-1}
 	 	EMULATOR_RAM_BASE_ADDRESS=${EMULATOR_RAM_BASE_ADDRESS::-1}
 
+		# this is a temp fix for building the emulator
+		cd $TOP_DIR/third_party/litex/litex/soc/cores/cpu/vexriscv/verilog/ext/VexRiscv
+		git am $TOP_DIR/0001-emulator-Use-external-hw-common.h-from-LiteX.patch
```

Comment:
Let's not include this at the top level.

Reply:
Moved to the newly-created folder `patches`.